services/auth/src/signaturevalidator.js

```diff
@@ -205,104 +205,59 @@ const createSignatureValidator = function(options) {
     'options.expandScopes must be a function');
   assert(options.monitor, 'options.monitor must be provided');
   assert(!options.nonceManager, 'nonceManager is not supported');
-  let loadCredentials = function(clientId, ext, callback) {
+
+  const loadCredentials = async (clientId, ext) => {
     // We may have two clientIds here: the credentialName (the one the caller
     // sent in the Hawk Authorization header) and the issuingClientId (the one
     // that signed the temporary credentials).
     let credentialName = clientId,
       issuingClientId = clientId;
 
-    (async () => {
```

Comment:
I wonder why this was this way before.

Reply:
It was an awkward mix of async/await and Hawk's old callback-based approach.